lib/js/styles/ItemColor.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+	<div>
+		<div v-for="item in items" :key="item.id" class="itemDefinition">
+			<div class="itemDefinition__id">{{ item.label }}</div>
+			<div class="itemDefinition__value">{{ item.value }}</div>
+			<div class="itemDefinition__color">
+				<span :style="{ background: item.value }" class="itemDefinition__tile" />
+			</div>
+		</div>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import '../../styles/components/tokens';
```

Comment:
are you sure, we're importing tokens here?

Reply:
Yes, because these are styles for visual component tokens (colors, typography) not tokens as variablels.

Reply:
It's confusing that scss file name and component name that are related, don't have anything in common. Do you think we should fix it?

Reply:
If you think that's confusing, sure. Changed.